Godeps/Godeps.json

```diff
@@ -435,82 +462,82 @@
 		},
 		{
 			"ImportPath": "github.com/vmware/govmomi",
-			"Comment": "v0.1.0-454-g9051bd6",
+			"Comment": "v0.6.2",
```

Comment:
It looks like vmware changed the tag to [0.6.2](https://github.com/vmware/govmomi/tree/v0.6.2). The actual commit is the same as it was previously but the `v0.1.0-454.g9051bd6` tag no longer exists in their repository.